src/utils/api/templates.js

```diff
@@ -28,4 +29,31 @@ export default class Templates {
         let constructed_url = Url.get('remove_template') + id + Url.getAccountParam();
         axios.post(constructed_url);
     }
+
+    static create(text) {
+        let request = {
+            account_id: store.state.account_id,
+            device_id: Api.generateId(),
+            text: Crypto.encrypt(text)
+        };
+
+        let constructed_url = Url.get('create_template');
+
+        const promise = new Promise((resolve) => {
+            axios.post(constructed_url, request, { 'Content-Type': 'application/json' })
+                .then(response => { resolve(response) });
+        });
+
+        return promise;
+    }
+
+    static update(id, text) {
+        let constructed_url = Url.get('update_template') + id + Url.getAccountParam() + "&text=" + text;
+        const promise = new Promise((resolve) => {
```

Comment:
Same here - prefer immediate return

Reply:
Hi, Thank you so much for reviewing the PR. For this suggestion, i think it is personal recommendation of how to return i.e. whether storing in a var or directly.